### PR TITLE
chore: added mandatory fields for task

### PR DESCRIPTION
**Pull request by CL-SHLOMIKONCHA**
| Question                | Answer                                                                          |
| ---------------- | -------------------------------------------------------------------------- |
| Bug fix         |✖                                                                        |
| New feature     | ✖                                                                       |
| Breaking change |✖                                                                        |
| Deprecations    | ✖                                                                        |
| Documentation   |✖                                                                        |
| Tests added     |✖                                                                        |
| Chore            | ✔                                                                     |

Related issues: #XXX , #XXX ...
Closes #XXX ...

Further  information:

### CODE REVIEW

File: src/serviceClients/storageClient.ts [hunk 1]
```diff
@@ -61,6 +61,7 @@ interface IGetJobResponse {
 }
 
 const jobType = 'Discrete-Tiling';
+const taskType = 'Discrete-Tiling';
```
Comment (asafMasa) [inline src/serviceClients/storageClient.ts:64]: move jobType and taskType to configuration

File: src/serviceClients/storageClient.ts [hunk 2]
```diff
@@ -75,6 +76,8 @@ export class StorageClient extends HttpClient {
   public async createLayerTasks(data: IngestionParams, zoomRanges: ITaskZoomRange[]): Promise<ITillerRequest[]> {
```
Comment (asafMasa) [inline src/serviceClients/storageClient.ts:76]: This method still return's a value (tasks)? why is it still in use?
https://github.com/MapColonies/discrete-overseer-creator/pull/60/files#diff-68293f52de110a02aca9266b7506f3c20d72501d92fb4d53064983a3e264d7fbR120

**Review by asafMasa — CHANGES_REQUESTED** [review]
Good Job 👍

File: src/serviceClients/storageClient.ts (hunk 2 above)
Comment (CL-SHLOMIKONCHA) [reply src/serviceClients/storageClient.ts]: fixed in 6bc91bf

File: src/serviceClients/storageClient.ts (hunk 1 above)
Comment (CL-SHLOMIKONCHA) [reply src/serviceClients/storageClient.ts]: fixed in 6bc91bf

**Review by asafMasa — APPROVED** [review]

**Pull request merged**